data/FreeBSD.yaml

```diff
@@ -1,3 +1,5 @@
 ---
+mcollective_choria::package_dependencies:
+  "mcollective": "present"
```

Comment:
@smortex is this still right for FreeBSD and OpenBSD? to install a package called `mcollective`? This is removed from https://github.com/choria-io/puppet-mcollective/pull/283

Reply:
Regarding FreeBSD: `rubygem-choria-mcorpc-support` was committed to the ports a few weeks ago, I am running this without issue, and I plan to drop the various legacy mcollective ports in early 2021… so I would recommend ensuring `rubygem-choria-mcorpc-support` is installed.

Regarding OpenBSD, I am not a user ATM, but [I only see a mcollective package here](https://ftp.openbsd.org/pub/OpenBSD/6.8/packages/amd64/), so sticking to `mcollective` is probably the safest.